packages/realm/src/Object.ts

```diff
@@ -252,13 +253,12 @@ export class RealmObject<T = DefaultObject> {
    */
   private readonly [INTERNAL_LISTENERS]!: ObjectListeners<T>;
 
-  // TODO: Find a way to bind this in
-  keys(): string[] {
-    throw new Error("This is expected to have a per-class implementation");
+  private keys(): string[] {
+    return Object.keys(this);
```

Comment:
This would result in an endless recursion if executed, but it's never called as [the `ClassHelper` provides a per-class implementation of the function](https://github.com/realm/realm-js/blob/bindgen/packages/realm/src/ClassMap.ts#L93-L101).